frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -228,13 +229,15 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
       pvc: pvcValidation,
       diskInterface: busValidation,
       url: urlValidation,
+      type: typeValidation,
     },
     isValid,
     hasAllRequiredFilled,
   } = validateDisk(resultDisk, resultVolume, resultDataVolume, resultPersistentVolumeClaim, {
     usedDiskNames,
     usedPVCNames,
     templateValidations,
+    canCreateCD,
```

Comment:
this will disqualify editing of vms with 2cdroms..
If we insist on such functionality it is better to have allowedDiskTypes passed

Reply:
won't this mean CDROM won't show in the select? this can be confusing.

Reply:
let's go with today's decision to not limit the amount of cdroms